src/platform/plugins/shared/field_formats/common/converters/numeral.ts

```diff
@@ -30,7 +30,9 @@ export abstract class NumeralFormat extends FieldFormat {
   abstract title: string;
 
   getParamDefaults = () => ({
-    pattern: this.getConfig!(`format:${this.id}:defaultPattern`),
+    // While this should be always defined, it is not guaranteed in testing that the function is available
+    pattern: this.getConfig?.(`format:${this.id}:defaultPattern`),
```

Comment:
Apparently a test was broken due to the force cast previously present who was throwing when `this.param` was looking for a missing parameter.